credential-generator/program-cert-pre-processor/src/main/scala/org/sunbird/job/programcert/functions/ProgramCertPreProcessorFn.scala

```diff
@@ -80,6 +80,7 @@ class ProgramCertPreProcessorFn(config: ProgramCertPreProcessorConfig, httpUtil:
             val leafNodeMap = mutable.Map[String, Int]()
 
             var isProgramCertificateToBeGenerated: Boolean = true;
+            var isSkippedEvent: Boolean = true;
```

Comment:
do rename the variable to "isFailedEvent" for better readability -- key the value to false.